src/components/descriptions/descContainer.tsx

```diff
@@ -23,142 +23,163 @@ interface descHeadingProps {
  * props : about, rule
  */
 
-export const DescHeading: React.FC<descHeadingProps> = ({ descType }) => {
-  return (
-    <>
-      <HeadingPresenter
-        heading={
-          {
-            about: "안녕하세요! 이곳은",
-            rule: "규칙",
-          }[descType]!
-        }
-        explanation={
-          {
-            about: "광대숲입니다!",
-            rule: "",
-          }[descType]!
-        }
-      />
-      {descType === "about" && (
-        <>
-          <p className={s.explanation}>
-            누구한테도 말하기 어려웠던 이야기부터 사소하고 조그만 이야기까지.
-          </p>
-          <hr className={s.greenLine} />
-        </>
-      )}
-    </>
-  );
+export const DescHeading: React.FC<descHeadingProps> = ({descType}) => {
```

Comment:
visit 'descType' often.

How about an integrated approach to this?
in the following manner

```ts
export const DescHeading: React.FC<descHeadingProps> = ({descType}) => {
    return (
        <>
            <HeadingPresenter
                heading={
                    {
                        about: "안녕하세요! 이곳은",
                        rule: "규칙",
                    }[descType]!
                }
                explanation={
                    {
                        about: "광대숲입니다!",
                        rule: "",
                    }[descType]!
                }
            />
            {descType === "about" && (
                <>
                    <p className={s.explanation}>
                        누구한테도 말하기 어려웠던 이야기부터 사소하고 조그만 이야기까지.
                    </p>
                    <hr className={s.greenLine}/>
                </>
            )}
        </>
    );
};
```